ParabolInc/parabol
# chore(metrics): Add Amplitude client-side page view tracking

# Description

Fixes #8794 

## Demo

✅ Logging in the Developer Tools console shows it successfully tracked the event:

<img width="1171" alt="Screenshot 2023-09-07 at 14 21 36" src="https://github.com/ParabolInc/parabol/assets/1879975/eb1113e9-56d0-4137-b17e-62ece0f838bd">


✅ `Loaded a Page` events show up in Amplitude Development environment:

<img width="1173" alt="Screenshot 2023-09-07 at 14 20 58" src="https://github.com/ParabolInc/parabol/assets/1879975/ed4556cc-9718-4d32-8d7e-c4a42096eb9a">

✅ Event stream for a user displays the event with proper properties. Notice the tracking library is from `amplitude-ts` (as compared to `Segment` that we have in production today)

<img width="1160" alt="Screenshot 2023-09-07 at 14 20 20" src="https://github.com/ParabolInc/parabol/assets/1879975/43e630f7-7948-4069-923f-2054e5021937">


## Testing scenarios
1. Go to any page
2. Open up console in Developer Tools and observer loggings indicating the events have been successfully tracked
3. Go to [this real-time Amplitude graph](https://app.amplitude.com/analytics/parabol/chart/13mww75e) and you should see new data points popping out

## Final checklist
